### PR TITLE
Some inline styles replaced by Tailwind CSS

### PR DESCRIPTION
**Pull request by drebarrera**

### CODE REVIEW

**Comment by leroyrosales** [comment]
@drebarrera Looks like there's some conflicts that need to be fixed.

File: components/About/Profile.js [hunk 1]
```diff
@@ -29,15 +12,15 @@ const LinkedIn = styled(FaLinkedin)`
 
 export default function Profile(props) {
   return (
-    <Aside>
-      <Image1 src={props.image} alt='Eddy' width={200} height={200} />
+    <div class="text-left max-w-[65ch] ">
+      <Image class="my-0 mx-auto rounded-[50%] block" src={props.image} alt='Eddy' width={200} height={200} />
```
Comment (leroyrosales) [inline components/About/Profile.js:16]: @drebarrera there's a few spots where you used class, replace those with classname

**Review by leroyrosales — CHANGES_REQUESTED** [review]

File: components/About/Profile.js (hunk 1 above)
Comment (drebarrera) [reply components/About/Profile.js]: All fixed!

**Comment by leroyrosales** [comment]
Had to fix a merge conflict, but otherwise looks good.

**Pull request merged**